validator/testcases/regex.py

```diff
@@ -257,6 +257,14 @@ def tests(self):
                 "primary purpose.",
                 "If this code does not change the behavior of the new tab "
                 "page, it may be ignored."),
+             "signing_help":
+                "Extensions may not programmatically override the new tab "
+                "page. If this code has another purpose, we nonetheless "
+                "recommend against testing URLs for these values, since "
+                "results can be unpredictable, and better options usually "
+                "exist. If you cannot avoid making these tests, please leave "
+                "this code unchanged, and it will be ignored in future "
+                "submissions.",
```

Comment:
In the other cases, I want to encourage developers to consider dropping the "features" that cause the flags. Here, I'd prefer that developers not change their code just to avoid the warning, so I chose different wording that I thought made that clearer.